setup.py

```diff
@@ -99,7 +99,8 @@ class Test(TestCommand):
 
     def run(self):
         """Run tests."""
-        cmd = "python3 -m pytest tests/ %s" % self.get_args()
+        cmd = "python3 -m pytest tests/ --cov-report term-missing"
```

Comment:
Nice. Great to have the `term-missing` added

Reply:
Related to kytos [issue](https://github.com/kytos-ng/kytos/issues/487).